exercises/acronym/example.jl

```diff
@@ -0,0 +1,5 @@
+#example.jl
+function myacronym(phrase)
```

Comment:
Perhaps just call it `acronym`. The `my` is only used in case the method name is already taken in Julia Base, which is not the case here.